### ISSUE TITLE
PropertyListReadStreamException: Unexpected end of plist

### ISSUE THREAD

**Issue by iulian0512** [electricbolt/dart-propertylistserialization#1]
Input plist  https://gist.github.com/iulian0512/fe8e8a1629346d459deadac9057c3c84

test code
 
```
 var xmlStr = File("/Users/ics/gitlab/mobileapp/ios/Runner/Info.plist").readAsStringSync();
 var lst = PropertyListSerialization.propertyListWithString(xmlStr) as List;


```

issue appears to happen if after the closing tag there is no newline

**Comment by electricbolt** [comment]
Hi good spotting. This is fixed in version 1.1.1
Please allow 30 minutes for dart.pub to finish processing this package version before attempting to upgrade.

**Comment by iulian0512** [comment]
apart from that very useful library, great work, thanks.
